lib/_stream_transform.js

```diff
@@ -92,7 +96,7 @@ function afterTransform(stream, er, data) {
     return stream.emit('error', new Error('no writecb in Transform class'));
```

Comment:
I left this alone to be safe. However, I haven't found any path to this code that doesn't involve calling the callback twice, or using internal APIs. If that's the case, the fix can be as simple as changing the error message on this line. I'll take a look in the morning.